cmd/neofs-node/config.go

```diff
@@ -969,8 +970,35 @@ func (c *cfg) configWatcher(ctx context.Context) {
 	}
 }
 
-// writeAppVersion writes app version as defined at compilation
-// step to the node's attributes.
-func writeAppVersion(c *cfg) {
+// writeSystemAttributes writes app version as defined at compilation
+// step to the node's attributes and an aggregated disk capacity
+// according to the all space on the all configured disks.
+func writeSystemAttributes(c *cfg) error {
+	// `Version` attribute
+
 	c.cfgNodeInfo.localInfo.SetAttribute("Version", misc.Version)
+
+	// `Capacity` attribute
+
+	var paths []string
+	for _, sh := range c.applicationConfiguration.EngineCfg.shards {
+		for _, storage := range sh.subStorages {
+			path := storage.path
+			paths = append(paths, path)
+
+			err := util.MkdirAllX(path, storage.perm)
+			if err != nil {
+				return fmt.Errorf("can not create (ensure it exists) dir by '%s' path: %w", path, err)
+			}
+		}
+	}
+
+	total, err := totalBytes(paths)
+	if err != nil {
+		return fmt.Errorf("calculating capacity on every shard: %w", err)
+	}
+
+	c.cfgNodeInfo.localInfo.SetCapacity(total / (1 << 30))
```

Comment:
i realize that these are drops and it is hardly possible, but how about setting `1GB` if everything is less?

Reply:
you mean preventing `0` announcement? well, mb not announce at all then?

@roman-khimov

Reply:
The it'd be an implicit zero? I don't think I have any preference here. 1GB when we don't have 1GB is not exactly correct. Zero capacity looks weird (is it a _storage_ node?). No capacity announced is also weird when every node has it. Explicit zero (the way it's now) is not the worst thing to have, at least we can see that the node has tried to calculate it and got zero (or it's configured this way explicitly which can also happen).

Reply:
at the same time 950M is not much far from 1.05G

in total, lets announce 0 for now (nodes will be chosen anyway), will revise in case of problems if any